File: src/ZKWeb.MVVMPlugins/MVVM.Common.Organization/src/Application/Dtos/WebsiteInfoOutputDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;
using ZKWeb.MVVMPlugins.MVVM.Common.Base.src.Application.Dtos;

namespace ZKWeb.MVVMPlugins.MVVM.Common.Organization.src.Application.Dtos {
	[Description("网站信息")]
	public class WebsiteInfoOutputDto : IOutputDto {
		[Description("ZKWeb版本")]
		public string ZKWebVersion { get; set; }
		[Description("ZKWeb完整版本")]
		public string ZKWebFullVersion { get; set; }
		[Description("使用内存")]
		public string MemoryUsage { get; set; }
		[Description("插件列表")]
		public IList<PluginInfoOutputDto> Plugins { get; set; }

		public WebsiteInfoOutputDto() {
			Plugins = new List<PluginInfoOutputDto>();
		}
	}
}

File: src/ZKWeb.MVVMPlugins/MVVM.Angular.Website/static/src/modules/generated_module/dtos/website-info-output-dto.ts
import { PluginInfoOutputDto } from './plugin-info-output-dto';

/** 网站信息 */
export class WebsiteInfoOutputDto {
    /** ZKWeb版本 */
    public ZKWebVersion: string;
    /** ZKWeb完整版本 */
    public ZKWebFullVersion: string;
    /** 使用内存 */
    public MemoryUsage: string;
    /** 插件列表 */
    public Plugins: PluginInfoOutputDto[];
}
